.github/workflows/nightly-test.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
         release: ["latest/edge"]
       fail-fast: false # TODO: remove once arm64 works
 
-    runs-on: ${{ matrix.arch == 'arm64' && ["self-hosted", "Linux", "ARM64", "jammy", "large"] || ["self-hosted", "Linux", "AMD64", "jammy", "large"] }}
+    runs-on: ${{ matrix.arch == 'arm64' && 'self-hosted-linux-arm64-jammy-large' || 'self-hosted-linux-amd64-jammy-large' }}
```

Comment:
Should we make this adjustment in our daily test as well?

Reply:
Not necessarily, we need it here because of the ``matrix.arch == 'arm64' &&`` check, which doesn't work when followed by a list.